### PR TITLE
Added two classes

### PR DESCRIPTION
**Pull request by Monsotya**

### CODE REVIEW

File: Sudoku.Application/ModifySudokuMethods.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Sudoku.Application
+{
+    public class ModifySudokuMethods
```
Comment (Yehukh) [inline Sudoku.Application/ModifySudokuMethods.cs]: Consider changing accessor to internal

Comment (Yehukh) [reply Sudoku.Application/ModifySudokuMethods.cs]: Also, I would like to rename the class name. Not self-describing :)

**Review by Yehukh — CHANGES_REQUESTED** [review]
Change the ModifySudokuMethods.cs

**Comment by Monsotya** [comment]
I need to make more changes

File: Sudoku.Application/MixSudoku.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Sudoku.Application
+{
+    public class MixSudoku
+    {
+        public void Transpose(int[][] matrix)
+        {
+        }
+		public void SwapRows(int[][] matrix)
```
Comment (Yehukh) [inline Sudoku.Application/MixSudoku.cs]: remove spaces in this function :)

File: Sudoku.Application/CreateSudoku.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace Sudoku.Application
+{
+    public class CreateSudoku
+    {
+        public CreateSudoku(int size, int complexity)
+        {
+        }
+
+        public void DeleteCells()
```
Comment (Yehukh) [inline Sudoku.Application/CreateSudoku.cs]: how many cells will be deleted. Should we specify this number as a parameter( for example defined by complexity)

**Review by Yehukh — CHANGES_REQUESTED** [review]

File: Sudoku.Application/MixSudoku.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Sudoku.Application
+{
+    public class MixSudoku
+    {
+        public void Transpose(int[][] matrix)
```
Comment (Yehukh) [inline Sudoku.Application/MixSudoku.cs]: Why we aren't returning this matrix back?

**Review by Yehukh — APPROVED** [review]

**Pull request merged**